.github/workflows/ci.yml

```diff
@@ -0,0 +1,16 @@
+name: CI
+
+on:
+  push:
+  pull_request:
+  workflow_dispatch:
+  # Every Tuesday at 11:20am UTC
+  schedule:
+    - cron: '20 11 * * 2'
+
+jobs:
+  ci:
+    name: CI
+    # Only run cron on the silverstripe account
+    if: (github.event_name == 'schedule' && startsWith(github.repository, 'silverstripe/')) || (github.event_name != 'schedule')
+    uses: silverstripe/gha-ci/.github/workflows/ci.yml@v1
```

Comment:
The .travis.yml file has some specific end-to-end suites to be run which do not seem to be represented here.

Reply:
I intentionally removed them.  For whatever strange historical reason, it was running admin tests, andd also running tests from the cms module.  The cms end-to-end tests automatically run on the cms repo - https://github.com/creative-commoners/silverstripe-cms/actions/runs/2614215668. The admin end-to-end tests run here https://github.com/creative-commoners/silverstripe-admin/actions/runs/2614027903